docs/documentation/concepts/index.md

```diff
@@ -6,29 +6,43 @@ layout: documentation
 
 # Concepts
 
-Eclipse SmartHome strictly differentiates between the physical view and the functional view on the system.
-While the physical view is required for setup, configuration, troubleshooting, etc., the functional side covers the information that matter to applications, such as user interfaces and automation logic.
+When first thinking about your home automation system, it may be helpful to bear in mind that there are two ways of thinking about or viewing your system; the physical view and the functional view.
 
-## Things, Channels, Items and Links
+The physical view will be familiar to you.
+This view focuses on the devices in your system, the connections between these devices (e.g. wires, Z-Wave, WiFi hardware), and other physical aspects of the system.
 
-**Things** are the entities that can be physically added to a system and which can potentially provide many functionalities at once. It is important to note that things do not have to be devices, but they can also represent a web service or any other manageable source of information and functionality.
-Things provide their functionality through a set of **Channels**. Channels are "passive" and can be regarded as a declaration of a Thing, what it can offer. It is up to the individual setup, which of the Channels are actively used through Items (see below).
+The functional view might be a new concept for you.
+The functional view focuses on how information about the devices, connections, and so on, is represented in user interfaces.
+The functional view includes focusing on how rules effect representations of physical devices in software.
+ Perhaps most important to you, the functional view focuses on how an action in a user interface affects the software associated with the physical device it represents.
```

Comment:
Does this start with an whitespace by intention?

Reply:
It is intended to be the start of a new paragraph focusing on the functional view.  The physical view was the focus of the previous paragraph.  Is that what you are seeing?  I get a failure when I try to render the page using jekyll, so I cannot see how the HTML looks.